### PR TITLE
pgadmin config examples

### LINKED ISSUE #249: document pgadmin
There are newer changes in latest version

* separate CRD for pgadmin
* OAUTH configuration is supported now

Need to add an example

### PR DESCRIPTION
**Pull request by rgherta**
Fixes #249

### CODE REVIEW

File: kustomize/pgadmin/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: postgres-operator
+
+resources:
+- pgadmin.yaml
```
Comment (benjaminjb) [inline kustomize/pgadmin/kustomization.yaml:7]: ⛏️ Can you add a newline at the end of this?

**Review by benjaminjb — APPROVED** [review]
LGTM -- I also wouldn't mind a README.md in this file that just said something like, "Available in v5.5.0 and above".

ETA: Or how about: "For more on this, please see the [docs]\(https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/namespace-scoped-pgadmin-4\)."

File: kustomize/pgadmin/kustomization.yaml (hunk 1 above)
Comment (rgherta) [reply kustomize/pgadmin/kustomization.yaml]: I think I did, can you double check? I also added the readme content

Comment (benjaminjb) [reply kustomize/pgadmin/kustomization.yaml]: Thx!

**Pull request merged**